src/Storages/ExternalStream/StorageExternalStream.h

```diff
@@ -22,6 +22,7 @@ class StorageExternalStream final : public shared_ptr_helper<StorageExternalStre
     void shutdown() override;
     bool supportsSubcolumns() const override;
     bool squashInsert() const noexcept override { return false; }
+    bool builtinSeekToSupport() const noexcept override { return true; }
```

Comment:
`supportsSeekTo` to keep the convention.

Reply:
`supportsSeekTo` is confusing. For example, when `supportsSeekTo` equals `false`, it sounds it does not support seek to at all, i.e. it should throw an error if seek to is used. If we want to keep the naming convention, we should probably name it `supportsNativeSeekTo`.

Reply:
I renamed it to `supportsNativeSeekTo`, please reopen this comment if you think that's not a proper name.